src/App.scss

```diff
@@ -1,11 +1,15 @@
 .App {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
remove `App.scss`,write all CSS handler into style folder,let name it correct,too